src/index.js

```diff
@@ -5,17 +5,23 @@ const validatePrTitle = require('./validatePrTitle');
 
 module.exports = async function run() {
   try {
-    const client = github.getOctokit(process.env.GITHUB_TOKEN);
     const {
       types,
       scopes,
       requireScope,
       wip,
       subjectPattern,
       subjectPatternError,
-      validateSingleCommit
+      validateSingleCommit,
+      githubServerUrl
     } = parseConfig();
 
+    const client = githubServerUrl
+      ? github.getOctokit(process.env.GITHUB_TOKEN)
+      : github.getOctokit(process.env.GITHUB_TOKEN, {
+        baseUrl: githubServerUrl
+      });
```

Comment:
Do you by chance know if the octokit client uses the default when `baseUrl` is `undefined`?

Maybe we could simplify it to just:

```suggestion
    const client = github.getOctokit(process.env.GITHUB_TOKEN, {
      baseUrl: githubServerUrl
    });
```

Reply:
I wasn't sure but I gave it a try and it does indeed uses the default :)